pkgs/by-name/xd/xdg-desktop-portal-luminous/package.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  meson,
+  ninja,
+  libclang,
+  clang,
+  xdg-desktop-portal,
+  slurp,
+  cairo,
+  pango,
+  libxkbcommon,
+  glib,
+  pipewire,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "xdg-desktop-portal-luminous";
+  version = "0.1.8";
+
+  src = fetchFromGitHub {
+    owner = "waycrate";
+    repo = pname;
```

`repo = pname` is an antipattern, it is better to be hardcoded instead

```suggestion
    repo = "xdg-desktop-portal-luminous";
```